docker-compose.yml

```diff
@@ -4,9 +4,19 @@ services:
     image: postgres:11.7-alpine
     volumes:
       - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
-    ports:
-      - "5432:5432"
     environment:
       POSTGRES_DB: i_have_i_need_development
       POSTGRES_USER: dev_local_user
-      POSTGRES_PASSWORD: password
\ No newline at end of file
+      POSTGRES_PASSWORD: password
```

Comment:
What do you think about moving these variables into a .env file? Its fairly common practice to try to remove any secrets from the system. 

[.env file](https://docs.docker.com/compose/environment-variables/#the-env-file)

So we could have the following file `$RAILS_ROOT/.env/development/database`
```
POSTGRES_USER=dev_local_user
POSTGRES_PASSWORD=password
POSTGRES_DB=beacon
```

Reply:
True, let's do in a separate PR. On the other hand I don't think it's a big deal to have dev/test secrets here